synapsis/src/main/resources/templates/pages/passwordResetForm.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://thymeleaf.org" xmlns:layout="https://www.ultraq.net.nz/thymeleaf/layout" layout:decorate="_layout">
+
+<div layout:fragment="content">
+
+    <form th:action="@{/passwordreset}" th:object="${registrationRequest}" method="post">
+
+        <div class="relative flex flex-col border p-8 justify-center mx-auto mt-10 shadow-xl w-1/2 mb-20">
+            <p class="font-thin text-3xl m-2 text-center text-grey">Enter your Email and your new Password</p>
+            <p th:text="${error}" class="font-light text-red-500"></p>
+            <input type="text" name = "email" id="email" th:field="*{email}" placeholder="Enter Email"
+               class="font-thin font-serif border border-grey mt-4 pl-3 h-14 content-center rounded-md hover:bg-gray-300" required/>
+            <input type="password" name ="password" id="password" th:field="*{password}" placeholder="Enter New Password"
```

Comment:
Would it be beneficial to have another input field for "Enter password again", or is that not necessary?

Reply:
I added the password toggle icon for that very purpose because people usually just copy paste their old passwords. So,  just toggling and seeing that they entered the correct password would be easier for them.

Reply:
I see. I guess that works as well also.